ui/src/app/app-routing.module.ts

```diff
@@ -105,7 +105,8 @@ const routes: Routes = [
                   breadcrumb: BreadcrumbType.Workspace,
                   helpContent: 'about'
                 }
-              }, {
+              },
```

Comment:
it was bugging me that I couldn't collapse stuff. I can revert this if you, esteemed reviewer, would prefer.